components/widgets/menu/menu_items/menu_cloud_trial.tsx

```diff
@@ -122,11 +135,9 @@ const MenuCloudTrial = ({id}: Props) => {
                 id='menu.cloudFree.tryEnterprise'
                 defaultMessage='Interested in a limitless plan with high-security features?'
             />
-
-            {/* Todo: modify this to open the see plans modal */}
             <a
                 className='open-see-plans-modal style-link'
-                onClick={() => null}
+                onClick={openPricingModal}
```

Comment:
this was already added: https://github.com/mattermost/mattermost-webapp/pull/10440/files#diff-67522593815da10d72efdaa9a1c9700c836c36833bf6cf4b0b8f950177e25634R129